src/com/boxboat/jenkins/library/Utils.groovy

```diff
@@ -65,6 +65,76 @@ class Utils implements Serializable {
         return format
     }
 
+    static def generateUUID(){
```

Comment:
the library tries to use explicit return types - helps with IDE hints
static UUID` instead of `static def`
this method may be unnecessary - see below

Reply:
Updated to `static String generateUUIDString`